README.adoc

```diff
@@ -1,50 +1,111 @@
-= agritrader

```

Comment:
I understand that the dockerize instructions for this case is different than the other projects. Is it okay then that the descriptions for how to use the dockerize script will vary for each project? I know I mentioned it before in a different pull request but now I'm wondering that maybe it isn't too big of a deal if they aren't consolidated?

Reply:
I think they shouldn't be consolidated. Even though it's more maintenance to upkeep all the files, I think it's too confusing to try to consolidate them. Also, if we have a product in the future that uses docker but doesn't use node at all, the instructions will be very very different and it wont' make sense to have a consolidated one.

Reply:
as long as we have a good readme explaining how to use it :)